src/mvc/ajaxRedirect.ts

```diff
@@ -68,12 +68,6 @@ export default class AjaxRedirect {
             url: url,
             type: 'GET',
             xhrFields: { withCredentials: true },
-            statusCode: {
```

Comment:
I found that we would get _0 code_  where there is any problem, ex: microservice is not running and we get _0 code_ while user should stay at the current page and redirect to the Login page where he is unauthorized.